client/js/components/map/publicdetail/controls/layerlist/DpPublicLayerListLayer.vue

```diff
@@ -21,6 +21,7 @@
       <button
         :class="prefixClass('btn--blank btn--focus w-3 text-left')"
         :aria-label="layer.attributes.name + ' ' + statusAriaText"
+        :data-cy="`publicLayerListLayer:${layerTitle.replace(/\s+/g, '-').toLowerCase()}`"
```

Comment:
do we have something more unique here? in the worst case maybe `id` ?

Reply:
When we use like `id`, then we have Hook like that `data-cy="publicLayerListLayer:1b4f6356-7222-11ec-92ee-b026282c0641"`

Reply:
Thats why I asked, if there is something better ;-)

Reply:
I will try to find something better :+1:

Reply:
I found a solution || idea:
We have in DpPublicLayerList `layerType`  Prop, then we can make name with or belongs tolayerType!
For example:
Kartenebenen ein/ausblenden:
- Kreis-,Amts-,Gemeindegrenzen: `data-cy="publicLayerListLayer:overlay:0"`
Grundkarten:
- Digitale Orthophotos (Luftbilder): `data-cy="publicLayerListLayer:base:0"`
- Web-Atlas: `data-cy="publicLayerListLayer:base:1"`
- Digitale Topographische Karte: `data-cy="publicLayerListLayer:base:2"`

Done in https://github.com/demos-europe/demosplan-core/pull/3187/commits/d1e708bd125c6f9a1a94b55278f9d2b1eeb98097